src/components/VmDetail/index.js

```diff
@@ -70,6 +76,9 @@ class VmDetail extends Component {
     const icon = icons.get(iconId)
     const disks = vm.get('disks')
 
+    const onToggleRenderDisks = () => { this.setState({ renderDisks: !this.state.renderDisks }) }
```

Comment:
why a stateful component? shouldn't we use Redux for *all* application state? E.g. a flag on the vm that would say whether to `renderDisks` or not.